src/analysis/box_integral/divergence_theorem.lean

```diff
@@ -72,8 +72,8 @@ begin
   integrate a function of the norm `≤ ε * diam I.Icc` over a box of volume
   `∏ j ≠ i, (I.upper j - I.lower j)`. Since `diam I.Icc ≤ c * (I.upper i - I.lower i)`, we get the
   required estimate.  -/
-  have Hl : I.lower i ∈ Icc (I.lower i) (I.upper i), from left_mem_Icc.2 (I.lower_le_upper i),
-  have Hu : I.upper i ∈ Icc (I.lower i) (I.upper i), from right_mem_Icc.2 (I.lower_le_upper i),
+  have Hl : I.lower i ∈ Icc (I.lower i) (I.upper i) := set.left_mem_Icc.2 (I.lower_le_upper i),
+  have Hu : I.upper i ∈ Icc (I.lower i) (I.upper i) := set.right_mem_Icc.2 (I.lower_le_upper i),
```

Comment:
Can you explain why these changes appeared in the PR description? I assume it's because `open set multiset` now causes a clash.